test/e2e/framework/statefulset_utils.go

```diff
@@ -481,13 +482,11 @@ func IsStatefulSetPodInitialized(pod v1.Pod) bool {
 }
 
 // NewStatefulSetPVC returns a PersistentVolumeClaim named name, for testing StatefulSets.
-func NewStatefulSetPVC(name string) v1.PersistentVolumeClaim {
+func NewStatefulSetPVC(c clientset.Interface, name string) v1.PersistentVolumeClaim {
```

Comment:
Make the storage class a parameter and retrieve the default prior to invoking this method. We shouldn't be calling retrieving the default storage class every time we create a new StatefulSet.